pycbc/workflow/minifollowups.py

```diff
@@ -197,6 +201,9 @@ def setup_single_det_minifollowups(workflow, single_trig_file, tmpltbank_file,
         assert(veto_segment_name is not None)
         node.add_input_opt('--veto-file', veto_file)
         node.add_opt('--veto-segment-name', veto_segment_name)
+    if statfiles is not None:
+        statfiles = statfiles.find_output_with_ifo(curr_ifo)
```

Comment:
I'm not sure this is strictly necessary (i.e. the stat files should be able to sort themselves out), however, I don't think it will cause a problem either.

Reply:
I guess it stops unnecessary parent->child relationships being created ...